### PR TITLE
Update README.md with a more verbose description

### PR DESCRIPTION
**Pull request by Gamester17**
Update README.md with a more verbose description for people new to ZHA Device Handlers in hope to maybe grasp its concept faster.

This is also closer match to the ZHA component description on Home Assistant site => https://www.home-assistant.io/components/zha/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # ZHA Device Handlers For Home Assistant
 
-ZHA Device Handlers are custom quirks implementations for [Zigpy](https://github.com/zigpy/zigpy), the library that provides the [Zigbee](http://www.zigbee.org) support for the [ZHA](https://www.home-assistant.io/components/zha/) component in [Home Assistant](https://www.home-assistant.io). These ZHA Device Handlers can be used to parse custom messages to and from Zigbee devices.
+ZHA Device Handlers are custom quirks implementations for [Zigpy](https://github.com/zigpy/zigpy), the library that provides the [Zigbee](http://www.zigbee.org) support for the [ZHA](https://www.home-assistant.io/components/zha/) component in [Home Assistant](https://www.home-assistant.io). 
+
+These ZHA Device Handlers are formost meant for exception and deviation handling can be used to parse custom messages to and from Zigbee devices. Zigbee devices that deviate from or do not fully conform to the standard specifications set by the Zigbee Alliance may require the development of custom ZHA Device Handlers (ZHA custom quirks handler implementation) to for all their functions to work properly with the ZHA component in Home Assistant.
```
Comment (dmulcahey) [inline README.md]: this: "These ZHA Device Handlers are formost meant for exception and deviation handling can be used to parse custom messages to and from Zigbee devices." reads poorly... How about:  "ZHA device handlers bridge the functionality gap created when manufacturers deviate from the ZCL specification."

Comment (Gamester17) [reply README.md]: How about: 

ZHA device handlers bridge the functionality gap created when manufacturers deviate from the ZCL specification, handling deviations and exceptions by parsing custom messages to and from Zigbee devices.

Or is that too verbose?

Comment (dmulcahey) [reply README.md]: That’s fine

Comment (Gamester17) [reply README.md]: Patched changed.

**Pull request merged**